app/models/kudo.rb

```diff
@@ -1,6 +1,7 @@
 class Kudo < ApplicationRecord
   belongs_to :receiver, class_name: 'Employee', inverse_of: :received_kudos
-  belongs_to :giver, class_name: 'Employee', inverse_of: :gived_kudos
+  belongs_to :giver, class_name: 'Employee', inverse_of: :given_kudos
+  belongs_to :company_value, inverse_of: :kudos
```

Comment:
no need for inverse_of if name of the association is matching the name of the class
```suggestion
  belongs_to :company_value
```